### ISSUE TITLE
No longer works starting from version-258fa44b42074cfc

### ISSUE THREAD

**Issue by kieldbg** [7ap/internal-studio-patcher#6]
The Roblox Studio Internal Patcher no longer works starting from version-258fa44b42074cfc (0.635.0.6350588) update of Roblox Studio. 

Internal privileges such as the ability to elevate the thread identity of the command bar greatly contributes the development of Drainer Drainer 123, and helps testing or playing around Roblox Lua functions that could be used to drain Robux from exploiters, or achieve arbitrary code execution.

Thank you,
![Picsart_24-02-03_16-04-13-953-1](https://github.com/user-attachments/assets/7e000fa9-844e-40a8-b6ae-0d2ae561021e)

**Comment by spearphishing** [comment]
super duper important

**Comment by 7ap** [comment]
Hi, as of `version-258fa44b42074cfc`, it does work.

```
internal-studio-patcher.exe version-258fa44b42074cfc/RobloxStudioBeta.exe
Patched in 239.0605ms.
```
